src/language/templates/visitors/nmodl_visitor.cpp

```diff
@@ -63,13 +63,17 @@ using namespace ast;
         auto symbol = std::string(order, '\'');
         printer->add_element(symbol);
     {% elif node.class_name == node_info.BINARY_EXPRESSION_NODE and child.varname == node_info.BINARY_OPERATOR_NAME %}
-        std::string op = node->get_op_cr().eval();
+        std::string op = node->get_op().eval();
         if(op == "=" || op == "&&" || op == "||" || op == "==")
             op = " " + op + " ";
         printer->add_element(op);
     {% else %}
         {% call guard(child.prefix, child.suffix) %}
+    {% if child.is_pointer_node %}
         node->get_{{ child.varname }}(){{ "->" if child.is_pointer_node else "." }}accept(*this);
```

Comment:
`if child.is_pointer_node` is superfluous.I suggest:
`node->get_{{ child.varname }}()->accept(*this);`

Reply:
or we remove the line before:

`{% if child.is_pointer_node %}`

Reply:
yes, sorry, you're right